src/main/java/org/cbioportal/persistence/mybatisclickhouse/StudyViewMyBatisRepository.java

```diff
@@ -260,6 +267,157 @@ public List<GenericAssayDataCountItem> getGenericAssayDataCounts(StudyViewFilter
         return mapper.getGenericAssayDataCounts(createStudyViewFilterHelper(studyViewFilterContext), genericAssayDataFilters);
     }
 
+    private Map<String, Map<String, GenePanelToGene>> _data = null;
+    
+    public Map<String, Map<String, GenePanelToGene>> getGenePanelsToGenes(String str){
+        
+        if (_data == null) {
```

Comment:
@haynescd this is a crude caching of the this data.  i know you'll find a better way